### PR TITLE
TD cell mixin

### PR DESCRIPTION
**Pull request by djbowman1**
Created mixing to apply to table cells/td so as to allow custom styling

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-grid/366)

### CODE REVIEW

File: vaadin-grid.html [hunk 1]
```diff
@@ -457,7 +459,8 @@
   .vaadin-grid-header th,
   .vaadin-grid-body td,
   .vaadin-grid-footer td {
-       padding: 0 24px 0 24px;
+    @apply(--vaadin-grid-row-cell);
```
Comment (billy-hardy) [inline vaadin-grid.html]: You should move this below the padding style, that way padding can be overridden in the mixin without the use of !important.

**Comment by djbowman1** [comment]
@jouni Is there a certain process that I have to go through to get this PR looked at by someone?

**Comment by jouni** [comment]
Hi! Sorry about the delay.

No special process, we just haven’t gotten around to looking at this yet. 

@Saulis, @manolo, @tomivirkki, @tehapo, @platosha what do guys think?

**Comment by Saulis** [comment]
Thanks for the PR!

Is the mixin meant to apply also for header and footer cells in addition to `tbody` ? 

<details><summary><i>Previously, jouni (Jouni Koivuviita) wrote…</i></summary>
> Hi! Sorry about the delay.
> 
> 
> 
> No special process, we just haven’t gotten around to looking at this yet. 
> 
> 
> 
> @Saulis, @manolo, @tomivirkki, @tehapo, @platosha what do guys think?
</details>


---

Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-grid/366#-:-KIRQ5ZlMA4XZuXdOPm3:-1562992716)*

**Comment by djbowman1** [comment]
@Saulis Yes it is meant to apply to the header and footer cells as well, sorry forgot to mention that in description

**Comment by Saulis** [comment]
:lgtm: Thanks!

<details><summary><i>Previously, djbowman1 wrote…</i></summary>
> @Saulis Yes it is meant to apply to the header and footer cells as well, sorry forgot to mention that in description
</details>


---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-grid/366#-:-KIWGV8xnfPRYT_0-HiC:-971291639)*

**Pull request merged**